sys/shell/commands/sc_cryptoauthlib.c

```diff
@@ -0,0 +1,213 @@
+/*
+ * Copyright (C) 2020 HAW Hamburg
+ *
+ * This file is subject to the terms and conditions of the GNU Lesser
+ * General Public License v2.1. See the file LICENSE in the top level
+ * directory for more details.
+ */
+
+/**
+ * @ingroup     sys_shell_commands
+ * @{
+ *
+ * @file
+ * @brief       Shell commands for the cryptoauthlib module
+ *
+ * @author      Lena Boeckmann <lena.boeckmann@haw-hamburg.de>
+ *
+ * @}
+ */
+#include <stdio.h>
+#include <stdint.h>
+#include "cryptoauthlib.h"
+
+void get_bin(char *result, uint8_t byte)
+{
+    for (int i = 0; i < 8; i++) {
+        result[i] = (byte << i & 0x80 ? '1' : '0');
+    }
+    result[8] = '\0';
+}
+
+int _read_config(int argc, char **argv)
```

Comment:
I would think that this output could be useful for automated tests or verification (in the future)? 
But therefore it must be easily parsable, i.e. have repeating structure. What I mean (or envision) would be a table like output, e.g.

```
0:3 00011100 00000000 00000000 00000000 Serial Number [0:3]
4:7 00110010 00000000 00000000 00000000 Revision Number
```

Not sure if this ideal, especially as the serial numbers second part is 5 bytes (weird layout of the config zone).
maybe octets are better

```
00011100 00000000 00000000 00000000 00110010 00000000 00000000 00000000 SN0 SN1 SN2 SN3 RN0 RN1 RN2 RN3
01110011 00100000 00110000 00011111 11111101 11111111 11111111 11111111 SN4 SN5 SN6 SN7 SN8 RSVD I2CE RSVD
```

Reply:
For better readability I'd rather keep the quartets. 
But if I name all the bytes like in your second suggestion, I can write the last byte of the serial number in the next line.